src/javascript/app/pages/user/accounts.js

```diff
@@ -180,6 +180,7 @@ const Accounts = (() => {
             additionalFunction: async () => {
                 localStorage.setItem('popup_action', action_map[action]);
                 SetCurrency.onLoad(onConfirmSetCurrency, redirect_to, all_fiat);
+                await SetCurrency.onLoad(onConfirmSetCurrency);
```

Comment:
had a conflict after pull upstream and waiting advice according to this part mamali

Reply:
Conflict resolved, I added `await to SetCurrency.onLoad(onConfirmSetCurrency, redirect_to, all_fiat);`